.gitignore

```diff
@@ -7,3 +7,6 @@ soft/python-3.7.4.zip
 soft/openjdk-11.0.2_windows-x64_bin.zip
 soft/postgresql-12.0-1-windows-x64-binaries.zip
 /input/anonymized_logs.zip
+result/
```

Comment:
It worth visiting our sessions. 
I asked to leave the results in as well as all the csv/txt files. 
It is required for quick demo of how the results look like.